samples/cpp/speech_sample/main.cpp

```diff
@@ -110,8 +111,7 @@ int main(int argc, char* argv[]) {
             model = core.read_model(FLAGS_m);
             if (!outputs.empty()) {
                 for (size_t i = 0; i < outputs.size(); i++) {
-                    auto output = model->add_output(outputs[i], ports[i]);
-                    output.set_names({outputs[i] + ":" + std::to_string(ports[i])});
```

Comment:
Why did you remove this string? Did you test the speech sample with the new output?

Reply:
Outputs of IR10 and IR11 models already have name with `:`

Reply:
But other layers - no.
Did you perform the following tests:
1) set any layers' output as results (--oname="layer_name:layer_port")
2) for the networks with several outputs get the result from the one only

Reply:
> I tested it with `-oname sigmoid5:0,Parameter:0`. The names of outputs was `sigmoid5:0` and `Parameter:0`

Because Parameter has tensor name too as input node :)